### PR TITLE
Add cancel jobs for closed/merged PRs

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Add cancel jobs for closed/merged PRs

## Links to any relevant issues

#258 

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

In my fork

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/cancel-jobs.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+jobs:
+  merge_job:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          ignore_sha: true
+          access_token: ${{ github.token }}
+          workflow_id: "bindings-nodejs.yml, bindings-python.yml, bindings-wallet-nodejs.yml, bindings-wasm.yml, build-and-test.yml, coverage.yml, lint.yml, private-tangle-tests.yml"
```
Comment (thibault-martinez) [inline .github/workflows/cancel-jobs.yml]: Is there no option to do it for them all? That sounds a bit annoying to maintain

Comment (Thoralf-M) [reply .github/workflows/cancel-jobs.yml]: There is, updated

**Review by Alex6323 — DISMISSED** [review]

**Comment by thibault-martinez** [comment]
What about https://github.com/styfle/cancel-workflow-action#advanced-pull-requests-from-forks ?

**Comment by Thoralf-M** [comment]
@thibault-martinez should also work now

**Review by qrayven — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**